native/dll/DShowPlayer-2/PlaybackGraph.cpp

```diff
@@ -84,25 +84,41 @@ JNIEXPORT void JNICALL Java_sage_DShowMediaPlayer_setVideoDecoderFilter0
 	slog((env, "DShowPlayer setVideoDecoderFilter0(%s) called\r\n", cName));
 	CComPtr<IBaseFilter> pFilter = NULL;
 	HRESULT hr;
-	if (!strcmp("SageTV MPEG Video Decoder", cName)|| !strcmp( "SageTV MPEG-2 Video Decoder", cName ) )
+	//if (!strcmp("SageTV MPEG Video Decoder", cName)|| !strcmp( "SageTV MPEG-2 Video Decoder", cName ) )

```

Comment:
Can you delete this whole block of commented out code? I'll go ahead and merge it after that (just trying to keep things cleaner now that everyone is sharing the code).